docs/install/docker/digital-ocean.md

```diff
@@ -4,9 +4,9 @@ navTitle: Docker on Digital Ocean
 
 # 1-Click Digital Ocean Droplet Install
 
-We have packaged the Docker Compose build of FlowFuse as a Digital Ocean Droplet. It can be found in the Digital Ocean Market Place [here](https://marketplace.digitalocean.com/apps/flowforge?refcode=fb23e438a477)
+We have packaged the Docker Compose build of FlowFuse as a Digital Ocean Droplet. It can be found in the Digital Ocean Market Place [here](https://marketplace.digitalocean.com/apps/flowfuse?refcode=fb23e438a477)
```

Comment:
```suggestion
We have packaged the Docker Compose build of FlowFuse as a Digital Ocean Droplet. It can be found in the [Digital Ocean Market Place](https://marketplace.digitalocean.com/apps/flowfuse?refcode=fb23e438a477).
```